org.eclipse.scanning.api/src/org/eclipse/scanning/api/device/models/DeviceWatchdogModel.java

```diff
@@ -16,7 +16,16 @@
 	{@literal <bean id="topupModel" class="org.eclipse.scanning.api.device.models.DeviceWatchdogModel">}
 	{@literal 	<property name="countdownName"          value="topup"/>}
 	{@literal 	<property name="cooloff"                value="4000"/>}
-	{@literal 	<property name="warmup"                 value="5000"/>}
+	{@literal 	<property name="warmup"                 value="5000"/>} 

```

Comment:
Where does the need for "warmup" and "cooloff" periods come from? I18 don't seem too bothered and would be happy to simply look for "time until next topup occurs" != 0. This would greatly simplify the code here. Are other beamlines wanting warmup and cooloff periods?